markdown/source_md/introduction.md

```diff
@@ -20,6 +20,7 @@ Although I bet that even if you don't have any significant programming experienc
 
 The channel #haskell on the Libera.Chat network is a great place to ask questions if you're feeling stuck.
 People there are extremely nice, patient and understanding to newbies.
+If IRC it not your cup of tea, https://discourse.haskell.org is a popular community forum with a section for learners.
```

Comment:
I'd recommend using markdown syntax to use `discourse.haskell.org` as link text.

Reply:
@memowe updated. Good to go? Or feel free to push into my branch.

Reply:
Awesome, thanks for your contribution! ❤️